examples/wallet/app/pages/InputKeys.js

```diff
@@ -59,15 +99,66 @@ export default ({ updateNodeSettings, setAccount, privateKey }: Props) => {
                 Go back
               </Button>
               <Button variant="primary" type="submit">
-                Initialize wallet
+                Initialize wallet Use key string
               </Button>
             </Row>
           </Form>
         </Container>
       </Tab>
       <Tab eventKey="mnemonic" title="Use mnemonic phrase">
         <Container>
-          <p>Under construction</p>
+          <Form onSubmit={handleSubmitMnemonic} className="mt-5">
+            <Form.Group>
+              <Form.Label>Wallet Seed:</Form.Label>
+              <Form.Control
+                required
+                type="text"
+                name="mnemonicPhrase"
+                placeholder="Enter your secret word phrase here to restore your vault. The phrase can have 12, 15, 18, 21 or 24 words."
+                value={newMnemonicPhrase}
+                onChange={event => setNewMnemonicPhrase(event.target.value)}
+              />
+              <Form.Text>
+                Example:
+                <br />
+                <code>
+                  decade panther require cruise robust mail gadget advice
+                  tonight post inner snack
+                </code>
+              </Form.Text>
+              <Form.Control
+                type="password"
+                name="mnemonicPassword"
+                placeholder="Secret password"
+                value={newMnemonicPassword}
+                onChange={event => setNewMnemonicPassword(event.target.value)}
+                className="mt-3"
+              />
+            </Form.Group>
+            <Row className="justify-content-between">
+              {/* TODO: bind this button */}
+              <Button variant="secondary" type="button">
+                Go back
+              </Button>
+              <Button variant="primary" type="submit">
+                Initialize wallet use mnemonic phrase
+              </Button>
+            </Row>
+          </Form>
+
+          <Modal show={showWrongMnemonicPhrase} onHide={handleClose}>
```

Comment:
https://blog.codinghorror.com/death-to-the-dialog-box/

I believe there are better ways to deal with this from a ux perspective(which could be combined).
- [  ] disable the button until the entered mnemonic is valid
- [  ] change the validation attributes of the input so it shows an error in the same input group if the mnemonic isn't valid.
This could even be configured to not bother the user while they is typing the mnemonic, but show up if the input loses focus, I think I saw bootstrap forms working like that in the past.